digitalpy/core/zmanager/impl/zmq_pusher.py

```diff
@@ -55,4 +63,7 @@
     def __setstate__(self, state):
         self.__dict__ = state
         self.pusher_context = zmq.Context()
-        self.pusher_socket = self.pusher_context.socket(zmq.PUSH)
+        self.pusher_socket = self.pusher_context.socket(zmq.PUSH)
+        
+        for connection in self.__pusher_socket_connections:
+            self.pusher_socket.connect(connection)
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>W292 no newline at end of file